src/python/turicreate/test/test_object_detector.py

```diff
@@ -114,30 +115,58 @@ def setUpClass(self):
         ## Answers
         self.opts = self.def_opts.copy()
         self.opts['max_iterations'] = 1
-
-        self.get_ans = {
-           '_model': lambda x: True,
-           '_class_to_index': lambda x: isinstance(x, dict),
-           '_training_time_as_string': lambda x: isinstance(x, str),
-           '_grid_shape': lambda x: tuple(x) == (13, 13),
-           'model': lambda x: x == self.pre_trained_model,
-           'anchors': lambda x: (isinstance(x, (list, tuple, np.ndarray)) and
-                                  len(x) > 0 and len(x[0]) == 2),
-           'input_image_shape': lambda x: tuple(x) == (3, 416, 416),
-           'batch_size': lambda x: x == 2,
-           'classes': lambda x: x == sorted(_CLASSES),
-           'feature': lambda x: x == self.feature,
-           'max_iterations': lambda x: x >= 0,
-           'non_maximum_suppression_threshold': lambda x: 0 <= x <= 1,
-           'training_time': lambda x: x > 0,
-           'training_iterations': lambda x: x > 0,
-           'training_epochs': lambda x: x >= 0,
-           'num_bounding_boxes': lambda x: x > 0,
-           'num_examples': lambda x: x > 0,
-           'training_loss': lambda x: x > 0,
-           'annotations': lambda x: x == self.annotations,
-           'num_classes': lambda x: x == len(_CLASSES),
-        }
+        if USE_CPP:
+            self.get_ans = {
+               'annotation_position': lambda x: isinstance(x, str),
```

Comment:
@nickjong please let me know if lambda checks for `annotation_position`, `annotation_scale`, `annotation_origin` , `training_average_precision_50`, `training_mean_average_precision_50`, `grid_height`, `grid_width`, `training_mean_average_precision`, `random_seed` and `training_average_precision` seem sufficient.